project/deps.sc

```diff
@@ -210,7 +210,7 @@ object Deps {
   def svm                       = ivy"org.graalvm.nativeimage:svm:$graalVmVersion"
   def swoval                    = ivy"com.swoval:file-tree-views:2.1.12"
   def testInterface             = ivy"org.scala-sbt:test-interface:1.0"
-  val toolkitVersion            = "0.4.0"
+  val toolkitVersion            = "0.5.0"
```

Comment:
I think we dropped Scala Native 0.4 with that version? Should we do anything about that?

Reply:
we dropped with 0.4.0, it's already handled.